automatic/firefox/README.md

```diff
@@ -17,7 +17,22 @@ Bringing together all kinds of awesomeness to make browsing better for you.
 
 ## Package Parameters
 
-- `l=<locale>` - Install given Firefox locale. For example `choco install Firefox --params "l=en-GB"`. See the [official page](https://releases.mozilla.org/pub/firefox/releases/latest/README.txt) for a complete list of available locales.
+- `l=<locale>` - Install given Firefox locale. See the [official page](https://releases.mozilla.org/pub/firefox/releases/latest/README.txt) for a complete list of available locales.
+
+Command-line options for installer configuration. See the [official page](https://firefox-source-docs.mozilla.org/browser/installer/windows/installer/FullConfig.html) for details and defaults.
+- `/InstallDirectoryPath=<path>`
```

Comment:
All other packages use `:`.

Reply:
That's correct and I agree. I used `=` because the package is already using it for the language settings `l=<locale>` and I didn't what to change that. If it's required it can be changed of course.

Reply:
I just found that we have to change it to use `:` anyway. The way it's currently implemented, the package parameters have to be followed by `=`. I.e. `NoDesktopShortcut` fails.